source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.md

```diff
@@ -81,31 +105,34 @@ parse error and MUST NOT do DNS resolution or contact hosts.
 It is an error to specify more than one host name in a connection string with the `mongodb+srv` protocol, and the driver
 MUST raise a parse error and MUST NOT do DNS resolution or contact hosts.
 
-A driver MUST verify that in addition to the `{hostname}`, the `{domainname}` consists of at least two parts: the domain
-name, and a TLD. Drivers MUST raise an error and MUST NOT contact the DNS server to obtain SRV (or TXT records) if the
-full URI does not consist of at least three parts.
+Node:
```

Comment:
What does 'Node:' mean?

Reply:
My mistake, must've ended up here on accident, good catch! Just removed it.